components/form-token-field/style.scss

```diff
@@ -189,3 +189,15 @@ input[type="text"].components-form-token-field__input {
 .components-form-token-field__suggestion-match {
 	color: $dark-gray-800;
 }
+
+.components-form-token__howto {
```

Comment:
Should we apply the built-in [`.screen-reader-text` class](https://make.wordpress.org/accessibility/2015/02/09/hiding-text-for-screen-readers-with-wordpress-core/) to avoid repeating it here?